pkg/sdk/poc/example/to_opts_optional_example_gen.go

```diff
@@ -0,0 +1,22 @@
+package example
+
+import "context"
+
+type ToOptsOptionalExamples interface {
+	Alter(ctx context.Context, request *AlterToOptsOptionalExampleRequest) error
+}
+
+// AlterToOptsOptionalExampleOptions is based on https://example.com.
+type AlterToOptsOptionalExampleOptions struct {
+	alter         bool                     `ddl:"static" sql:"ALTER"`
+	IfExists      *bool                    `ddl:"keyword" sql:"IF EXISTS"`
+	name          DatabaseObjectIdentifier `ddl:"identifier"`
+	OptionalField *OptionalField           `ddl:"keyword"`
+	RequiredField RequiredField            `ddl:"keyword"`
+}
```

Comment:
nit: shouldn't we add space between structs?

Reply:
+1

Reply:
This is a current behavior of the generator that is not related to the to_opts_mapping 
but I agree that there should be spaces between the structs